node/node.go

```diff
@@ -1563,6 +1563,15 @@ func (app *App) startAPIServices(ctx context.Context) error {
 		if err := app.grpcPublicServer.Start(); err != nil {
 			return err
 		}
+		logger.With().Info("public grpc service started",
+			log.String("address", app.Config.API.PublicListener),
+			log.Array("services", log.ArrayMarshalerFunc(func(encoder log.ArrayEncoder) error {
+				for svc := range publicSvcs {
```

Comment:
This is not really a serious problem, but iterating over maps here may produce different results each time. This may cause some minor issues when processing logs, e.g. trying to find differences between successful and failed test run logs after filtering out timestamps etc. It would be a bit better to use `maps.Keys` to get the list of map's keys and then sort them

Reply:
Updated code accordingly